doc/isorms.xml

```diff
@@ -54,6 +54,68 @@
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="RMSIsoByTriple">
+  <ManSection>
+    <Oper Name = "RMSIsoByTriple" Arg = "R1, R2, triple"/>
+    <Oper Name = "RZMSIsoByTriple" Arg = "R1, R2, triple"/>
+    <Returns>An isomorphism.</Returns>
+    <Description>
+      If <A>R1</A> and <A>R2</A> are isomorphic Rees 0-matrix semigroups then
+      <C>RZMSIsoByTriple</C> returns the isomorphism between <A>R1</A> and
+      <A>R2</A> defined by <A>triple</A>, which should be a list consisting of
+      the following:
+
+      <List>
+        <Item>
+          <C><A>triple</A>[1]</C> should be a permutation describing an
+          isomorphism from the graph of <A>R1</A> to the graph of <A>R2</A>,
+          i.e. it should satisfy
+          <C>OnDigraphs(RZMSDigraph(<A>R1</A>), <A>triple</A>[1])
+             = RZMSDigraph(<A>R2</A>)</C>.
+        </Item>
+        <Item>
+          <C><A>triple</A>[2]</C> should be an isomorphism from the underlying
+          group of <A>R1</A> to the underlying group of <A>R2</A> (see
+          <Ref Attr="UnderlyingSemigroup"
+               Label="for a Rees 0-matrix semigroup"
+               BookName="ref"/>).
+        </Item>
+        <Item>
+          <C><A>triple</A>[3]</C> should be a list of elements from the
+          underlying group of <A>R2</A>.  If the
+          <Ref Attr="Matrix" BookName="ref"/> of <A>R1</A> has <M>m</M> columns
+          and <M>n</M> rows, then the list should have length <M>m + n</M>,
+          where the first <M>m</M> entries should correspond to the columns of
+          <A>R1</A>'s matrix, and the last <M>n</M> entries should correspond to
+          the rows.
```

Comment:
More specific here, refer to Howie's theorem and the fact that `triple[3]{[1 .. m]}`  should consist of the u_i from Theorem 3.4.1 and `triple[3]{[m + 1 ... n]}` consist of the `v_lambda ^ -1`, subject to checking that I haven't got `u` and `v` muddled up.